### PR TITLE
support plan replayer for multi sqls

### PR DESCRIPTION
**Pull request by Yisaer**
Signed-off-by: yisaer <disxiaofei@163.com>

### What is changed, added or deleted? (Required)

support plan replayer for multi sqls

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1430,6 +1430,13 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
     - 事务从异步提交或一阶段提交到两阶段提交的回退信息
     - 遇到的错误
 
+### `tidb_last_plan_replayer_token` <span class="version-mark">从 v6.3.0 版本开始引入</span>
+
+- 作用域：SESSION
+- 是否持久化到集群：否
+- 类型：字符串
+- 此变量用于获取当前会话中最后一个 plan replayer dump 的结果。这是一个只读变量。
```
Comment (xuyifangreeneyes) [inline system-variables.md:1438]: 可以说明一下该变量是只读变量

Comment (Yisaer) [reply system-variables.md:1438]: 已经声明了

Comment (shichun-0415) [reply system-variables.md:1438]: FD jira issue 是哪个呢？

File: sql-plan-replayer.md [hunk 2]
```diff
@@ -60,11 +60,38 @@ MySQL [test]> plan replayer dump explain select * from t;
 +------------------------------------------------------------------+
 | Dump_link                                                        |
 +------------------------------------------------------------------+
-| replayer_single_JOGvpu4t7dssySqJfTtS4A==_1635750890568691080.zip |
+| replayer_JOGvpu4t7dssySqJfTtS4A==_1635750890568691080.zip |
 +------------------------------------------------------------------+
 1 row in set (0.015 sec)
 ```
 
+你同样可以通过 `tidb_last_plan_replayer_token` 这个会话变量来获取上一次 plan replayer dump 执行的结果。
```
Comment (shichun-0415) [inline sql-plan-replayer.md:68]:
```suggestion
你同样可以通过 [`tidb_last_plan_replayer_token`](/system-variables.md#tidb_last_plan_replayer_token-从-v630-版本开始引入) 这个会话变量来获取上一次 plan replayer dump 执行的结果。
```

File: sql-plan-replayer.md [hunk 3]
```diff
@@ -60,11 +60,38 @@ MySQL [test]> plan replayer dump explain select * from t;
...
+
+```sql
+mysql> select @@tidb_last_plan_replayer_token;
++-----------------------------------------------------------+
```
Comment (shichun-0415) [inline sql-plan-replayer.md:72]:
```suggestion
select @@tidb_last_plan_replayer_token;
```

```sql
+-----------------------------------------------------------+
```

File: sql-plan-replayer.md [hunk 4]
```diff
@@ -60,11 +60,38 @@ MySQL [test]> plan replayer dump explain select * from t;
...
+| @@tidb_last_plan_replayer_token                           |
++-----------------------------------------------------------+
+| replayer_Fdamsm3C7ZiPJ-LQqgVjkA==_1663304195885090000.zip |
++-----------------------------------------------------------+
+1 row in set (0.00 sec)
+```
+
+对于多条 sql 的情况，你可以通过文件的方式来获取 plan replayer dump 的结果，多条 sql 在文件中以 `;` 进行分割。
```
Comment (shichun-0415) [inline sql-plan-replayer.md:80]:
```suggestion
对于多条 sql 的情况，你可以通过文件的方式来获取 plan replayer dump 的结果，多条 sql 在文件中以 `;` 进行分隔。
```

File: sql-plan-replayer.md [hunk 5]
```diff
@@ -60,11 +60,38 @@ MySQL [test]> plan replayer dump explain select * from t;
...
+
+```sql
+mysql> plan replayer dump explain 'sqls.txt';
+Query OK, 0 rows affected (0.03 sec)
```
Comment (shichun-0415) [inline sql-plan-replayer.md:84]:
```suggestion
plan replayer dump explain 'sqls.txt';
```

```sql
Query OK, 0 rows affected (0.03 sec)
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

File: sql-plan-replayer.md [hunk 6]
```diff
@@ -60,11 +60,38 @@ MySQL [test]> plan replayer dump explain select * from t;
 +------------------------------------------------------------------+
 | Dump_link                                                        |
 +------------------------------------------------------------------+
-| replayer_single_JOGvpu4t7dssySqJfTtS4A==_1635750890568691080.zip |
+| replayer_JOGvpu4t7dssySqJfTtS4A==_1635750890568691080.zip |
```
Comment (Oreoxmt) [inline sql-plan-replayer.md:63]: L108 需要修改为：

```
curl http://127.0.0.1:10080/plan_replayer/dump/replayer_JOGvpu4t7dssySqJfTtS4A==_1635750890568691080.zip > plan_replayer.zip
```

File: system-variables.md (hunk 1 above)
Comment (Oreoxmt) [inline system-variables.md:1438]:
```suggestion
- 这个变量是一个只读变量，用于获取当前会话中最后一个 `PLAN REPLAYER dump` 的结果。
```

File: sql-plan-replayer.md [hunk 7]
```diff
@@ -60,11 +60,38 @@ MySQL [test]> plan replayer dump explain select * from t;
...
+1 row in set (0.00 sec)
+```
+
+对于多条 sql 的情况，你可以通过文件的方式来获取 plan replayer dump 的结果，多条 sql 在文件中以 `;` 进行分割。
+
+```sql
+mysql> plan replayer dump explain 'sqls.txt';
+Query OK, 0 rows affected (0.03 sec)
+
+mysql> select @@tidb_last_plan_replayer_token;
```
Comment (Oreoxmt) [inline sql-plan-replayer.md:86]:
```suggestion
mysql> SELECT @@tidb_last_plan_replayer_token;
```

**Review by shichun-0415 — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**